Scripts/RemediationScripts/Remediate-EnableEncryptionInTransitForStorageAccounts.ps1

```diff
@@ -0,0 +1,691 @@
+                $stgWithDisableHTTPS | Export-CSV -Path "$($folderpath)\StorageWithDisableHTTPS.csv" -NoTypeInformation
+                Write-Host "Path: $($folderPath)\StorageWithDisableHTTPS.csv" -ForegroundColor $([Constants]::MessageType.Update)
+                write-host $([Constants]::SingleDashLine)
+               
+                if (-not $Force)
+                {
+                    Write-Host "Do you want to enable secure transfer on the storage account(s)? " -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+                    $userInput = Read-Host -Prompt "(Y|N)"
+    
+                    if($userInput -ne "Y")
+                    {
+                        Write-Host "Secure transfer will not be enabled for any of the storage account. Exiting..." -ForegroundColor $([Constants]::MessageType.Info)
+                        break
+                    }
+                }
+                else
+                {
+                    Write-Host "'Force' flag is provided. Secure transfer will be enabled on the storage account without any further prompts." -ForegroundColor $([Constants]::MessageType.Warning)
+                }
+   
+                write-host $([Constants]::DoubleDashLine)
+                Write-Host "[step 6 of 6] :Enable 'secure transfer' for the Storage Account(s) of subscription."
+
+                #Storage Account passed from remediation
+                $remediationSuccess = @() 
+                #Storage Account failed from remediation
+                $remediationFailure = @()
+    
+                Write-Host "Enabling 'secure transfer' on [$($totalStgWithEnableHTTPS)] Storage Account(s)..."
+                $stgWithDisableHTTPS = $stgWithDisableHTTPS | Sort-Object -Property "ResourceGroupName"
+                $stgWithDisableHTTPS | ForEach-Object {
+                    try
+                    {   
+                        $output = Set-AzStorageAccount -ResourceGroupName $_.ResourceGroupName -Name $_.StorageAccountName -EnableHttpsTrafficOnly $true -ErrorAction SilentlyContinue
+                        if($output -ne $null)
+                        {
+                            $remediationSuccess += $_ 
+                        }   
+                        else
+                        {   
+                            Write-Host "Remediation is not successful on [$($_.StorageAccountName)] : [$($_.ResourceGroupName)]" -ForegroundColor $([Constants]::MessageType.Warning) 
+                            $remediationFailure += $_
+                        }
+                    }        
+                    catch
+                    {
+                        Write-Host "Error occurred while remediating [$($_.StorageAccountName)] : [$($_.ResourceGroupName)]" -ForegroundColor $([Constants]::MessageType.Error) 
+                        write-Host  "Error: $($_)" -ForegroundColor $([Constants]::MessageType.Error)
```

Comment:
`write-Host` to be replaced with `Write-Host`.

Reply:
Done